src/status_im/mailserver/core.cljs

```diff
@@ -416,27 +416,61 @@
 (fx/defn update-mailserver-topics
   "TODO: add support for cursors
   if there is a cursor, do not update `last-request`"
```

Comment:
I think it is worth mentioning the error scenario in the description too

Reply:
this function is not called at all if there is an error

Reply:
my bad I just realized I forgot to remove the error code as I handled it outside of this function